core/modules/node/node.install

```diff
@@ -102,6 +102,13 @@ function node_schema() {
         'not null' => TRUE,
         'default' => 0,
       ),
+      'published_on' => array(
```

Comment:
This is going to confuse the heck out of everyone next to the `status` column. I'm suggesting `published_date` or `publish_date` (since its not yet happened).